src/mame/drivers/rastersp.cpp

```diff
@@ -188,13 +257,52 @@ void rastersp_state::machine_start()
 	m_dsp->space(AS_PROGRAM).install_read_handler(0x809923, 0x809923, read32smo_delegate(*this, FUNC(rastersp_state::dsp_speedup_r)));
 	m_dsp->space(AS_PROGRAM).install_write_handler(0x809923, 0x809923, write32smo_delegate(*this, FUNC(rastersp_state::dsp_speedup_w)));
 #endif
+
+	save_item(NAME(m_irq_status));
+	save_item(NAME(m_dlba));
+	save_pointer(NAME(m_paletteram), 0x8000);
+	save_item(NAME(m_palette_number));
+	save_item(NAME(m_speedup_count));
+	save_item(NAME(m_tms_io_regs));
+	save_item(NAME(m_port2_data));
+	save_item(NAME(m_last_watchdog));
+	save_item(NAME(m_left_volume));
+	save_item(NAME(m_right_volume));
+	save_item(NAME(m_interrupt_mask));
 }
 
+void fbcrazy_state::machine_start()
+{
+	rastersp_state::machine_start();
+
+	if (!m_edc_tb_timer)
+		m_edc_tb_timer = machine().scheduler().timer_alloc(timer_expired_delegate(FUNC(fbcrazy_state::edc_tb_timer), this));
```

Comment:
You can assume timers don’t exist when starting unless you’re expecting a derived class to throw a missing dependencies exception *after* calling the base implementation.

Reply:
OK, hadn't realised that. I've made the change locally and will commit with all the changes which are going to be needed.